libsoftwarecontainer/component-test/devicenodegateway_componenttest.cpp

```diff
@@ -97,3 +97,60 @@ TEST_F(DeviceNodeGatewayTest, TestOverwriteDeviceFails) {
     checkMode.wait();
     ASSERT_TRUE(checkMode.isSuccess());
 }
+
+struct testSetup
+{
+    const std::string firstConfiguration;
+    const std::string secondConfiguration;
```

Comment:
This is a good point, the quick-launch feature might not require more than two sequences at the moment, but our support for this in general is not so far limited to two and I don't think this should be either.